deepmd/entrypoints/test.py

```diff
@@ -870,7 +870,7 @@ def test_polar(
         arrays with results and their shapes
     """
     data.add(
-        "polarizability" if not atomic else "atomic_polarizability",
```

Comment:
It seems a breaking change. What's the reason to introduce it? Besides, lots of codes still use `atomic_`, such as those in `data/`, `examples/`, `doc/`, `tf/loss/`.

Reply:
> It seems a breaking change. What's the reason to introduce it? Besides, lots of codes still use `atomic_`, such as those in `data/`, `examples/`, `doc/`, `tf/loss/`.

currently some fitting nets use `atomic_***` as the label while others use `atom_***`, when calculating stat with atomic labels, we have to introduce hardcoded keys depending on the fitting nets, which will be hard to maintain.

Reply:
maybe we can keep using `atomic_***` in tf, if we separate the data.

Reply:
Could we support both? Just need to implement an alias feature in the data requirement.

Reply:
I have reverted these changes. Will address this issue when migrating polar bias.